ajax/libs/gitalk/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "gitalk",
+  "filename": "gitalk.min.js",
+  "version": "1.1.4",
+  "description": "A comment plugin base on Github issues",
```

Comment:
Maybe use [`Gitalk is a modern comment component based on Github Issue and Preact.`](https://github.com/gitalk/gitalk#gitalk) is better than `A comment plugin base on Github issues`.
What do you think?

Thanks